pkg/client/client.go

```diff
@@ -62,7 +62,17 @@ func NewClient(token, apiEndpoint string) (runtimeclient.Client, error) {
 }
 
 func NewClientFromRestConfig(config *rest.Config) (runtimeclient.Client, error) {
-	config.Insecure = true
+	tc, err := config.TransportConfig()
```

Comment:
I assume this is being fixed separately in https://github.com/kubesaw/ksctl/pull/55 and this PR will adopt the solution arrived upon there?

Reply:
yup correct! I've just merged master branch!